second.rb

```diff
@@ -0,0 +1,24 @@
+person_hash = {}
+
+first_name = gets.chomp
+last_name = gets.chomp
+age = gets.chomp
+street_address = gets.chomp
+city = gets.chomp
+state = gets.chomp
+
+person_hash[:first_name] = first_name
+person_hash[:last_name] = last_name
+person_hash[:age] = age
+person_hash[:city] = city
+person_hash[:street_address] = street_address
+person_hash[:state] = state
+
+puts person_hash
+puts person_hash.keys
+puts person_hash.values
+
+person_hash[:first_name] = person_hash[:first_name].capitalize
+person_hash[:state] = person_hash[:state].capitalize
```

Comment:
❗ `#capitalize` capitalizes the first character of a word, but the directions want all characters of `:state` to all be uppercase. There is a different Ruby [string](https://ruby-doc.org/3.2.2/String.html) method for this.

> _Change the state as stored in the person hash to upper case._